lang/cs/Org.Apache.REEF.Network/Elastic/Operators/Physical/Default/DefaultOneToN.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.Threading;
+using System.Collections.Generic;
+using Org.Apache.REEF.Network.Elastic.Topology.Physical.Default;
+using System;
+using Org.Apache.REEF.Network.Elastic.Comm.Impl;
+using Org.Apache.REEF.Utilities.Logging;
+using Org.Apache.REEF.Utilities.Attributes;
+using Org.Apache.REEF.Network.Elastic.Operators.Physical.Enum;
+using Org.Apache.REEF.Network.Elastic.Comm;
+
+namespace Org.Apache.REEF.Network.Elastic.Operators.Physical.Default
+{
+    /// <summary>
+    /// Generic implementation of a group communication operator where one node sends to N.
+    /// </summary>
+    /// <typeparam name="T">The type of message being sent.</typeparam>
+    [Unstable("0.16", "API may change")]
+    public abstract class DefaultOneToN<T>
+    {
+        private static readonly Logger Log = Logger.GetLogger(typeof(DefaultOneToN<>));
+
+        internal readonly OneToNTopology _topology;
+        internal volatile PositionTracker _position;
+
+        private readonly bool _isLast;
+        private bool _cleanDisposal;
+
+        /// <summary>
+        /// Creates a new one to N operator.
+        /// </summary>
+        /// <param name="id">The operator identifier</param>
+        /// <param name="level">The checkpoint level for the operator</param>
+        /// <param name="isLast">Whether this operator is the last in the pipeline</param>
+        /// <param name="topology">The operator topology layer</param>
+        internal DefaultOneToN(int id, bool isLast, OneToNTopology topology)
+        {
+            OperatorId = id;
+            _isLast = isLast;
+            _topology = topology;
+            _position = PositionTracker.Nil;
+            _cleanDisposal = false;
+
+            OnTaskRescheduled = new Action(() =>
+            {
+                _topology.JoinTopology();
+            });
```

Comment:
no need to wrap it (twice!) into lambda and then `Action` -- this is just
```C#
OnTaskRescheduled = _topology.JoinTopology;
```

Reply:
ah! Fixed.